src/test/java/resources/mojaloop.feature

```diff
@@ -77,7 +78,8 @@ Feature: Mojaloop test
     Then I call the deposit account endpoint with command "deposit" for amount 10 for "payee"
 
     Then I add "payer" to als
-    Then I add "payee" to als
+#    payee/payee2/payee3
```

Comment:
why payee is comented? are we doing this test only for payee3 ?
can you add this test description accordingly?

Reply:
The test can be done for any of the payees, that's what the comment intends to tell, this test does not intend to test Multi-tenancy. But these changes help people refer to the test cases and use the same step definitions used in this test.

Reply:
Should I make the comments a bit more clear?

Reply:
suggestion is to keep it in a data table so that when needed these can be run for multiple payees.

Reply:
Scenario Outline was the right approach to implement your suggestion. I have updated the test.

Reply:
![image](https://github.com/fynarfin/ph-ee-integration-test/assets/140409886/36c97fc3-a1d8-4792-a990-79757b922050)